crates/sui-e2e-tests/tests/reconfiguration_tests.rs

```diff
@@ -2,29 +2,48 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use futures::future::join_all;
+use move_core_types::ident_str;
 use rand::rngs::OsRng;
-use std::collections::BTreeSet;
 use std::sync::Arc;
 use std::time::Duration;
 use sui_core::consensus_adapter::position_submit_certificate;
+use sui_json_rpc_types::ObjectChange;
 use sui_json_rpc_types::SuiTransactionBlockEffectsAPI;
 use sui_macros::sim_test;
 use sui_node::SuiNodeHandle;
+use sui_protocol_config::ProtocolVersion;
 use sui_protocol_config::{Chain, ProtocolConfig};
-use sui_swarm_config::genesis_config::{ValidatorGenesisConfig, ValidatorGenesisConfigBuilder};
+use sui_swarm_config::genesis_config::{
+    AccountConfig, ValidatorGenesisConfig, ValidatorGenesisConfigBuilder,
+};
 use sui_test_transaction_builder::{make_transfer_sui_transaction, TestTransactionBuilder};
 use sui_types::base_types::SuiAddress;
+use sui_types::effects::TransactionEffects;
 use sui_types::effects::TransactionEffectsAPI;
+use sui_types::effects::TransactionEvents;
 use sui_types::error::SuiError;
-use sui_types::governance::MIN_VALIDATOR_JOINING_STAKE_MIST;
+use sui_types::governance::{
+    VALIDATOR_LOW_POWER_PHASE_1, VALIDATOR_MIN_POWER_PHASE_1, VALIDATOR_VERY_LOW_POWER_PHASE_1,
+};
+use sui_types::programmable_transaction_builder::ProgrammableTransactionBuilder;
 use sui_types::sui_system_state::{
     get_validator_from_table, sui_system_state_summary::get_validator_by_pool_id,
     SuiSystemStateTrait,
 };
-use sui_types::transaction::{TransactionDataAPI, TransactionExpiration, VerifiedTransaction};
+use sui_types::transaction::{
+    Command, TransactionDataAPI, TransactionExpiration, VerifiedTransaction,
+};
+use sui_types::SUI_SYSTEM_PACKAGE_ID;
 use test_cluster::{TestCluster, TestClusterBuilder};
 use tokio::time::sleep;
 
+use sui_types::transaction::Argument;
+use sui_types::transaction::ObjectArg;
+use sui_types::transaction::ProgrammableMoveCall;
+
+const DEFAULT_GAS_AMOUNT: u64 = 30_000_000_000_000_000; // 30M Sui
```

Comment:
This constant should be available somewhere else.

Reply:
Thanks! Yes, it is available in genesis_config, I've changed to import it instead.